An OSTFL Grant announcement post (#278)
sunpy/sunpy.org: posts/2021/2021-10-08_nasa_osftl_grant.rst

> .. post:: 8th October 2021

Author: ```suggestion
.. post:: 3rd November 2021
```

   :author: Stuart Mumford
   :tags: sunpy
   :category: Update

SunPy Awarded NASA Grant
========================

> **As part of the NASA "Open Source Tools, Frameworks, and Libraries" program the SunPy proposal "Strengthening the Foundations of the SunPy Ecosystem" has been funded for the next three years.**

Reviewer: ```suggestion
**As part of the NASA "Open Source Tools, Frameworks, and Libraries" program, the SunPy proposal "Strengthening the Foundations of the SunPy Ecosystem" has been funded for the next three years.**
```

> This proposal was submitted in January 2021, and aims to provide funding for both the core `sunpy` package and wider goals of the project.

Reviewer: ```suggestion
This proposal was submitted in January 2021, and sought funding for both the core `sunpy` package and wider goals of the project.
```

> After `some early community input <https://github.com/sunpy/sunpy-project/issues/9>`__ Albert Shih took on leading writing the proposal, with the funds being awarded through NASA GSFC.

Reviewer: ```suggestion
After `early community input was collected <https://github.com/sunpy/sunpy-project/issues/9>`__, Albert Shih led the proposal itself, with NASA GSFC as the primary institution.
```

> `The proposal <https://docs.google.com/document/d/1_gf1HM7iIUVqgHAdDUFQfCUHzHkrEFUTJZP8O3PEoqw>`__ focuses on three main areas:

Reviewer: ```suggestion
`The proposed effort <https://docs.google.com/document/d/1_gf1HM7iIUVqgHAdDUFQfCUHzHkrEFUTJZP8O3PEoqw>`__ focuses on three main areas:
```

   As part of this work templates for setting up new pacakges will be improved, documentation will be written, and new automation and monitoring bots will be developed to ease the burden on the SunPy maintainers.

2. Augmenting science-enabling functionality.

   This section aims to improve the tools provided by the core package to better support future challenges in solar physics data.
   The two main components of this are supporting multi-point observations with improvements to the `sunpy.coordinates` package, and improving the support for very large datasets in `sunpy` and sponsored packages by leveraging the scaling and parallelism tools available from `Dask <https://dask.org>`__.

3. Providing training and outreach

   This section will improve example galleries across both `sunpy` core and affiliated packages, develop training materials for solar physics graduate students, and perform outreach activities with instrument teams and package developers.

What will the money pay for?
----------------------------

> The money awarded to this proposal will pay for development time:

Reviewer: ```suggestion
The money awarded to this proposal will pay for:
```

>   * Albert Shih will manage the project and be funded to spend some time in his role as `sunpy.coordinates` maintainer.

Reviewer: ```suggestion
  * Albert Shih will manage the project and augment parts of `sunpy.coordinates`.
```

>   * Stuart Mumford will be funded to work on infrastructure for all sponsored and affiliated packages as discussed in `sunpy/sunpy-project#2 <https://github.com/sunpy/sunpy-project/issues/2>`__.

Reviewer: ```suggestion
  * Stuart Mumford will work on infrastructure for all sponsored and affiliated packages as discussed in `sunpy/sunpy-project#2 <https://github.com/sunpy/sunpy-project/issues/2>`__.
```

>   * Albert Shih will manage the project and augment parts of `sunpy.coordinates`.
>   * Stuart Mumford will work on infrastructure for all sponsored and affiliated packages as discussed in `sunpy/sunpy-project#2 <https://github.com/sunpy/sunpy-project/issues/2>`__.
>   * Will Barnes will work on the training/outreach and large-dataset-support components of the project.

Author: ```suggestion
* Albert Shih will manage the project and augment parts of `sunpy.coordinates`.
* Stuart Mumford will work on infrastructure for all sponsored and affiliated packages as discussed in `sunpy/sunpy-project#2 <https://github.com/sunpy/sunpy-project/issues/2>`__.
* Will Barnes will work on the training/outreach and large-dataset-support components of the project.
```

>   * A 70% FTE role will be funded based at NASA GSFC focusing on the outreach and large dataset support components of the project.

Reviewer: ```suggestion
  * Will Barnes will work on the training/outreach and large-dataset-support components of the project.
```

Finally
-------

This is a major milestone for SunPy - it is the first significant amount of funding dedicated to directly working on the long term sustainability of the project.

> Everyone currently involved in the project is hopeful that this funding will help to significantly grow SunPy; both the core package and the ecosystem of other packages we want to grow around it.

Reviewer: ```suggestion
This funding will help to significantly grow SunPy, both the core package and the ecosystem of associated packages.
```

.. post:: 8th October 2021
   :author: Stuart Mumford
   :tags: sunpy
   :category: Update

> SunPy Awarded a NASA Grant
> ==========================

Reviewer 2: ```suggestion
SunPy Awarded NASA Grant
========================
```

**As part of the NASA "Open Source Tools, Frameworks, and Libraries" program the SunPy proposal "Strengthening the Foundations of the SunPy Ecosystem" has been funded for the next three years.**

> This proposal was submitted in January 2021, it aims to provide funding for both the core `sunpy` package and the larger goals of the project as a whole.

Reviewer 2: ```suggestion
This proposal was submitted in January 2021, and aims to provide funding for both the core `sunpy` package and wider goals of the project.
```

After `some early community input <https://github.com/sunpy/sunpy-project/issues/9>`__ Albert Shih took on leading writing the proposal, with the funds being awarded through NASA GSFC.
`The proposal <https://docs.google.com/document/d/1_gf1HM7iIUVqgHAdDUFQfCUHzHkrEFUTJZP8O3PEoqw>`__ focuses on three main areas:

1. Improving the technical infrastructure.

>    This section focuses on the tools that the project as a whole needs as we adopt more coordinated and affiliated packages.

Reviewer 2: ```suggestion
   This focuses on the tools that the wider project needs to scale up as we adopt more coordinated and affiliated packages.
```

>    To support multiple active projects with a common development workflow more tooling and infrastructure is needed, both for the maintainers of the packages and developers new to writing affiliated packages.

Reviewer 2: ```suggestion
   To support multiple active projects with a common development workflow, more tooling and infrastructure is needed, both for maintainers of existing packages and developers new to writing affiliated packages.
```

>    As part of this work the package template(s) will be improved, documentation will be written, and new bots and monitoring will be developed to ease the maintenance burden on the SunPy maintainers.

Reviewer 2: ```suggestion
   As part of this work templates for setting up new pacakges will be improved, documentation will be written, and new automation and monitoring bots will be developed to ease the burden on the SunPy maintainers.
```
Also, documentation for what?

2. Augmenting science-enabling functionality.

   This section aims to improve the tools provided by the core package to better support future challenges in solar physics data.

>    The two main components of this are supporting multi-point observations with improvements to the `sunpy.coordinates` package, and improving the support for very large datasets in `sunpy` and sponsored packages by increasing support for `Dask <https://dask.org>`__.

Reviewer 2: ```suggestion
   The two main components of this are supporting multi-point observations with improvements to the `sunpy.coordinates` package, and improving the support for very large datasets in `sunpy` and sponsored packages by leveraging the scaling and parallelism tools available from `Dask <https://dask.org>`__.
```

3. Providing training and outreach

>    This section will improve the example gallery's of `sunpy` and the affiliated packages, develop training materials for solar physics grad students, and outreach with instrument teams and package developers.

Reviewer 2: ```suggestion
   This section will improve example galleries across both `sunpy` core and affiliated packages, develop training materials for solar physics graduate students, and perform outreach activities with instrument teams and package developers.
```

What will the money pay for?
----------------------------

The money awarded to this proposal will pay for development time:

>   * Albert Shih will be funded to spend some time in his role as `sunpy.coordinates` maintainer.

Reviewer 2: ```suggestion
  * Albert Shih will manage the project and be funded to spend some time in his role as `sunpy.coordinates` maintainer.
```

  * Stuart Mumford will be funded to work on infrastructure for all sponsored and affiliated packages as discussed in `sunpy/sunpy-project#2 <https://github.com/sunpy/sunpy-project/issues/2>`__.

>   * Lastly, it will fund a 70% FTE role based at NASA GSFC focusing on the outreach and large dataset support components of the project.

Reviewer 2: Are there reasons that Will isn't namechecked here?

Reviewer 2: ```suggestion
  * A 70% FTE role will be funded based at NASA GSFC focusing on the outreach and large dataset support components of the project.
```

Finally
-------

> On a personal note, this is a major milestone for SunPy, this is a significant amount of funding to be dedicated to directly working for the long term good of the project.

Reviewer 2: ```suggestion
This is a major milestone for SunPy - it is the first significant amount of funding dedicated to directly working on the long term sustainability of the project.
```

> I am hopeful that this funding and the time it will enable us to spend on the project will help to significantly grow SunPy; both the core package and the ecosystem of other packages we want to grow around it.

Reviewer 2: ```suggestion
Everyone currently involved in the project is hopeful that this funding will help to significantly grow SunPy; both the core package and the ecosystem of other packages we want to grow around it.
```

> Also, I am just really excited to be able to call working on SunPy my job for a few years 🥳.

Reviewer 2: ```suggestion
```
I think it sounds better coming from "the project" instead of a personal point of view? (this isn't Stuart's personal blog after all 😆 )
